### PR TITLE
fix various nits

### PR DESCRIPTION
**Pull request by djfritz**

### CODE REVIEW

File: configuration/certificates.md [hunk 1]
```diff
@@ -92,11 +92,11 @@ systemctl restart gravwell_searchagent.service
 
 ### Making browsers trust the self-signed certificate
 
-Browsers will throw up a warning if a certificate is not signed by a recognized root CA. However, we can make the browser trust our certificate by installing it manually.
+Browsers will issue a warning if a certificate is not signed by a recognized root CA. However, you can make the browser trust our certificate by installing it manually.
```
Comment (floren) [inline configuration/certificates.md:95]: "we" is used throughout the document, so I think we should use it here.

File: quickstart/downloads.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Downloads
 
-Attention: The Debian repository is more easily maintained than these standalone installers and is the recommended method of installation. See the [quickstart instructions](#!quickstart/quickstart.md).
+Attention: The Debian and RHEL repositories are more easily maintained than these standalone installers and is the recommended method of installation. See the [quickstart instructions](#!quickstart/quickstart.md).
```
Comment (floren) [inline quickstart/downloads.md:3]: "is the recommended" should be "are the recommended"

File: search/langfind/langfind.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ tag=reddit json Body | langfind -e Body | count by lang | sort by count desc | t
 
 ### Supported Options
 
-* `-e <arg>`: The “-e” option operates on an enumerated value instead of on the entire record. For example, a pipeline that performed language analysis on HTTP payloads would be `tag=pcap ipv4.DstPort==80 tcp.Payload | langfind -e Payload`
+* `-e <arg>`: The “-e” option operates on an enumerated value instead of on the entire record. For example, a pipeline that performed language analysis on HTTP payloads would be `tag=pcap ipv4.DstPort==80 tcp.Payload | langfind -e Payload`.
```
Comment (floren) [inline search/langfind/langfind.md:15]: Note that when this is displayed, there will not be single ticks. It's too easy for a user to copy-paste `langfind -e Payload.` if they're not paying attention.

File: configuration/certificates.md (hunk 1 above)
Comment (djfritz) [reply configuration/certificates.md:95]: "you" is used 8 times in that document, "we" is used 3.

File: search/langfind/langfind.md (hunk 3 above)
Comment (djfritz) [reply search/langfind/langfind.md:15]: @floren PTAL

**Comment by djfritz** [comment]
@floren PTAL

**Pull request merged**